monai/transforms/inverse_batch_transform.py

```diff
@@ -73,17 +75,23 @@ def __init__(
                 if set to `None`, use the `num_workers` of the transform data loader.
             detach: whether to detach the tensors. Scalars tensors will be detached into number types
                 instead of torch tensors.
+            pad_batch: when the items in a batch indicate different batch size,
+                whether to pad all the sequences to the longest.
+                If False, the batch size will be the length of the shortest sequence.
+            fill_value: the value to fill the padded sequences when `pad_batch=True`.
```

Comment:
Could you please help add the default behavior when `fill_value=None`?

Thanks.